app/src/main/java/org/mozilla/focus/FocusApplication.kt

```diff
@@ -60,10 +72,21 @@ class FocusApplication : LocaleAwareApplication() {
         customTabSessions.observeForever(TelemetrySessionObserver())
     }
 
+    fun loadExperiments() {
+        val experimentsFile = File(filesDir, EXPERIMENTS_JSON_FILENAME)
+        val experimentSource = KintoExperimentSource(
+                EXPERIMENTS_BASE_URL, EXPERIMENTS_BUCKET_NAME, EXPERIMENTS_COLLECTION_NAME)
+        fretboard = Fretboard(experimentSource, FlatFileExperimentStorage(experimentsFile))
+        launch(CommonPool) {
+            fretboard.loadExperiments() // load current settings from disk
```

Comment:
Do we want to put this on an IO context?

Context: the CommonPool has a limited number of threads so doing several blocking calls (I'm assuming loading settings from disk is blocking) can delay others from using the CommonPool who probably don't expect their execution to be delayed. See https://mail.mozilla.org/pipermail/sync-dev/2018-July/001667.html for a better explanation.

Reply:
LOL. I made the same complaint earlier about another PR. I think we should create our own thread pool on startup.

Reply:
Technically, there is no IO CoroutineDispatcher. That's from RxJava. I just created my own.